components/resources/brave_components_strings.grd

```diff
@@ -801,6 +801,7 @@
       <part file="tor_strings.grdp" />
       <part file="brave_news_strings.grdp" />
       <part file="commands.grdp" />
+      <part file="app_management_additional.grdp" />
```

Comment:
I'm wondering if this grdp should instead be inserted in the same place that the settings additional strings are - `app/brave_generated_resources.grd`